request/builder.js

```diff
@@ -759,6 +688,5 @@ module.exports = {
   ContractsListingRequestBuilder,
   LedgerValidationRequestBuilder,
   ContractExecutionRequestBuilder,
-  RequestProofRegistrationRequestBuilder,
-  ContractExecutionRequestWithAssetProofsBuilder,
+  ExecutionValidationRequestBuilder,
 };
```

Comment:
Do we miss the `OrderExecutionRequestBuilder` implementation?

Reply:
Good catch! Thank you! The ordering request for the auditor uses the same GRPC request as the ledger. So, I should remove the unnecessary codes related to `OrderExecutionRequestBuilder`. I'll fix it.